tests/model_explainability/trustyai_service/service/multi_ns/test_trustyai_service_multi_ns.py

```
@@ -0,0 +1,119 @@
+    @pytest.mark.parametrize("ns_index", [0, 1])
+    def test_upload_data_to_trustyai_service_multiple_ns(
+            self,
+            ns_index: int,
+            admin_client,
+            current_client_token,
+            trustyai_service_with_pvc_storage_multi_ns,
+            gaussian_credit_model_multi_ns,
+            isvc_getter_token_multi_ns,
+            model_namespaces,
+    ) -> None:
+            tai = trustyai_service_with_pvc_storage_multi_ns[ns_index]
```

Flake8 (7.2.0)

[error] over-indented

(E117)